cyvlfeat/fisher/cyfisher.pyx

```diff
@@ -0,0 +1,67 @@
+# distutils: language = c
+# Copyright (C) 2007-12 Andrea Vedaldi and Brian Fulkerson.
+# All rights reserved.
+
+# This file is modified from part of the VLFeat library and is made available
+# under the terms of the BSD license.
+import numpy as np
+cimport numpy as np
+cimport cython
+
+# Import the header files
+from cyvlfeat._vl.fisher cimport vl_fisher_encode
+
+@cython.boundscheck(False)
+cpdef cy_fisher(np.ndarray[float, ndim=2, mode='c'] X,
+                np.ndarray[float, ndim=2, mode='c'] MEANS,
+                np.ndarray[float, ndim=2, mode='c'] COVARIANCES,
+                np.ndarray[float, ndim=1, mode='c'] PRIORS,
+                bint Normalized,
+                bint SquareRoot,
+                bint Improved,
+                bint Fast,
+                bint Verbose):
+
+    vl_size numClusters = MEANS.shape[1];
+    vl_size dimension = MEANS.shape[0];
+    vl_size numData = X.shape[1];
+    int flags = 0;
+
+    if Normalized:
+        flags |= VL_FISHER_FLAG_NORMALIZED
```

Comment:
These flags are presumably declared somewhere in a vlfeat header, and need to be declared as the `vl_fisher_encode` was, in that header file. They then need to be `cimported` here.